src/models/enums/MessageCodes.ts

```diff
@@ -15,4 +15,9 @@ export enum MessageCodes {
 	FAILED_FETCHING_SESSION = "FAILED_FETCHING_SESSION",
 	INVALID_PERSONAL_DETAILS = "INVALID_PERSONAL_DETAILS",
 	EXPIRED_SESSION = "EXPIRED_SESSION",
+	FAILED_VALIDATING_ACCESS_TOKEN_REQUEST_BODY = "FAILED_VALIDATING_ACCESS_TOKEN_REQUEST_BODY",
```

Comment:
Do you think we could make this more generic so it can be reused? Something like `FAILED_VALIDATING_REQUEST_BODY`? Since we can see the service in the logs anyway